client/my-sites/plan-features/index.jsx

```diff
@@ -88,27 +93,31 @@ PlanFeatures.propTypes = {
 	available: PropTypes.bool,
 	onUpgradeClick: PropTypes.func,
 	// either you specify the plan prop or isPlaceholder prop
-	plan: React.PropTypes.string,
-	isPlaceholder: PropTypes.bool
+	plan: PropTypes.string,
+	isPlaceholder: PropTypes.bool,
+	isInSignup: PropTypes.bool
```

Comment:
The main difference is basically that the site specific plans data, has extra info like discount pricing.